frameworks/helloworld/tests/test_multiservice_dynamic.py

```diff
@@ -48,10 +49,14 @@ def test_add_deploy_restart_remove():
     assert not service['uninstall']
 
     sdk_plan.wait_for_plan_status(config.SERVICE_NAME, 'deploy', 'COMPLETE', multiservice_name=svc1)
+    old_task_id = sdk_tasks.get_task_ids(config.SERVICE_NAME, 'hello-world')[0]
```

Comment:
Parameters here may need to be `'marathon', config.SERVICE_NAME` (`SERVICE_NAME` == `hello-world` already)

Could also log the list of task ids returned by `get_task_ids` before taking the first one from the list, in case none or multiple are found or something